package.json

```diff
@@ -7,13 +7,21 @@
         "build": "next build",
         "start": "next start",
         "export": "next build && next export",
-        "lint": "next lint",
-        "format": "prettier --write \"**/**/*.{js,ts,tsx}\"",
+        "lint": "eslint --cache",
```

Comment:
next lint runs eslint under the hook, also we are already using the estlint-next config in the eslintrc file so we can swap the commands out without changing anything. the reason we need to use eslint --cache is so that lint-staged can pass multiple files to eslint as well as the ability to use --cache for speeding things up